src/material/schematics/ng-update/test-cases/v12/misc/theming-api-v12.spec.ts

```diff
@@ -716,4 +716,22 @@ describe('v12 theming API migration', () => {
       `@include mat.mdc-button-theme();`,
     ].join('\n'));
   });
+
+  it('should not drop imports of prebuilt styles', async () => {
+    writeLines(THEME_PATH, [
+      `@import '~@angular/material/prebuilt-themes/indigo-pink.css';`,
+      `@import '~@angular/material/theming';`,
+      `@import '~@angular/cdk/overlay-prebuilt.css';`,
+      `@include mat-core();`,
+    ]);
+
+    await runMigration();
+
+    expect(splitFile(THEME_PATH)).toEqual([
+      `@use '~@angular/material' as mat;`,
+      `@import '~@angular/material/prebuilt-themes/indigo-pink.css';`,
+      `@import '~@angular/cdk/overlay-prebuilt.css';`,
```

Comment:
It should be possible to migrate these to the following
```suggestion
      `@use '@angular/material/prebuilt-themes/indigo-pink.css';`,
      `@use '@angular/cdk/overlay-prebuilt.css';`,
```
Or is that something that you are depending upon the sass-migrator to do?

Reply:
I believe that imports of `.css` files are still supposed to go through `@import`.

Reply:
I thought that too at first, but found that they work fine with `@use` as long as the deprecated `sass-loader` feature of using `~` is not used.

If you'd seen an official statement from the Sass team to the otherwise, please do let me know.